recipes/biobear/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2023 Trent Hauck
```

Comment:
The license is in the tarball. You can delete this file.

Reply:
done in [be768f7](https://github.com/conda-forge/staged-recipes/pull/23350/commits/be768f77a4a4acccb2f31ce32cf536f5c9857c60)